integration_tests/doc_example_driver.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env sh
+
+# This script drives httpie to interact with a local development ltd-keeper
+# instance. The output can be be used for examples in the docs.
+#
+# Development server prep
+# 1. Delete any existing ltd-keeper-dev.sqlite
+# 2. Run the dev server ./run.py
+# 3. Get an auth token
+#        http --auth user:pass GET http://localhost/token
```

Comment:
This uses httpie. You should probably link to it here and the README.

https://github.com/jkbrzt/httpie